aspnetcore/tutorials/first-mvc-app/working-with-sql.md

```diff
@@ -33,31 +37,36 @@ The ASP.NET Core [Configuration](xref:fundamentals/configuration/index) system r
 
 ---
 
-When the app is deployed to a test or production server, an environment variable can be used to set the connection string to a production SQL Server. See [Configuration](xref:fundamentals/configuration/index) for more information.
+When the app is deployed to a test or production server, an environment variable can be used to set the connection string to a production SQL Server. For more information, see [Configuration](xref:fundamentals/configuration/index) for more information.
 
 # [Visual Studio](#tab/visual-studio)
 
 ## SQL Server Express LocalDB
 
-LocalDB is a lightweight version of the SQL Server Express Database Engine that's targeted for program development. LocalDB starts on demand and runs in user mode, so there's no complex configuration. By default, LocalDB database creates *.mdf* files in the *C:/Users/{user}* directory.
+LocalDB:
 
-* From the **View** menu, open **SQL Server Object Explorer** (SSOX).
+* Is a lightweight version of the SQL Server Express Database Engine.
+* Targeted for program development.
+* Starts on demand and runs in user mode, so there's no complex configuration.
+* By default creates *.mdf* assets in the *C:/Users/{user}* directory.
```

Comment:
```suggestion
* By default creates *.mdf* files in the *C:/Users/{user}* directory.
```

Reply:
I had a note that either you or Scott suggested we use "assets" when referring to files in this way earlier.  I scratch the note.